test/controllers/YourIncomeCalculationControllerSpec.scala

```diff
@@ -220,19 +221,28 @@ class YourIncomeCalculationControllerSpec extends PlaySpec
   val personService: PersonService = mock[PersonService]
   val employmentService = mock[EmploymentService]
   val taxAccountService = mock[TaxAccountService]
+  val paymentsService = mock[PaymentsService]
```

Comment:
Do we really need to mock the payments service, if possible use the concrete type